src/components/BackLink/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import {
+  Link as AscLink,
+  Icon,
+  Typography,
+  themeColor,
+  themeSpacing,
+} from '@datapunt/asc-ui';
+import { ChevronLeft } from '@datapunt/asc-assets';
+
+const LinkLabel = styled(Typography).attrs({
+  $as: 'span',
+})`
+  font-size: 16px;
+  color: ${themeColor('primary')};
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+`;
+
+const Chevron = styled(ChevronLeft)`
+  display: inline-block;
+`;
+
+const StyledLink = styled(AscLink)`
+  &:hover {
+    * {
+      color: ${themeColor('secondary')};
+    }
+
+    path {
+      fill: ${themeColor('secondary')} !important;
+    }
+
+    text-decoration: none;
+  }
+`;
+
+const StyledIcon = styled(Icon)`
+  margin: 0 ${themeSpacing(2)} 0 0 !important;
+  display: inline-block;
+
+  svg > path {
+    fill: ${themeColor('primary')} !important;
+  }
+`;
+
+/**
+ * Component that renders a Link with a left chevron
+ * To be used on detail pages for navigating back to its corresponding overview page
+ */
+const BackLink = ({ className, children, to }) => (
+  <StyledLink className={className} $as={Link} to={to} data-testid="backlink">
+    <StyledIcon size={12}>
+      <Chevron color={themeColor('primary')} />
+    </StyledIcon>
+    <LinkLabel>{children}</LinkLabel>
+  </StyledLink>
+);
+
+BackLink.defaultProps = {
+  className: '',
+};
+
+BackLink.propTypes = {
+  /** The BackLink label */
+  children: PropTypes.node.isRequired,
```

Comment:
that is incorrect comment isn't it?

Reply:
The `children` prop can be considered the component's label, but I guess the term `label` is a bit ambiguous. I'll change it.